frontend/src/views/planosSetoriais/PlanosSetoriaisLista.vue

```diff
@@ -100,49 +100,51 @@ if (!lista.length) {
             {{ item.prefeito }}
           </td>
           <td>{{ item.ativo ? 'Sim' : 'Não' }}</td>
-          <td class="tr">
-            <SmaeLink
-              v-if="item.pode_editar"
-              class="tprimary mr1 tipinfo left"
-              :to="{
-                name: `${route.meta.entidadeMãe}.planosSetoriaisEditar`,
-                params: { planoSetorialId: item.id }
-              }"
-            >
-              <svg
-                width="20"
-                height="20"
-              ><use xlink:href="#i_edit" /></svg>
-            </SmaeLink>
+          <td>
+            <span class="flex justifycenter g1">
+              <SmaeLink
+                v-if="item.pode_editar"
+                class="tprimary tipinfo left"
+                :to="{
+                  name: `${route.meta.entidadeMãe}.planosSetoriaisEditar`,
```

Comment:
SmaeLink não precisa de entidadeMae
```suggestion
                  name: '.planosSetoriaisEditar',
```